transcendence-app/seeds/001_seed_users.ts

```diff
@@ -30,40 +30,53 @@ const createUserWithPassword = async (username: string, avatarId: string) => {
     email: `${username}@${username}.com`,
     fullName: username,
     avatarId: avatarId,
-    password: hashedPassword,
+    password: hashedPassword
   };
 };
 
 export async function seed(knex: Knex): Promise<void> {
-  // Deletes ALL existing entries and local files
-  await knex('users').del();
-  const appDataPath = configService.get('TRANSCENDENCE_APP_DATA') as string;
+  const adminUsername = await configService.get("WEBSITE_OWNER_USERNAME");
+  const [admin, _] = await knex("users")
+    .select("*")
+    .where("username", adminUsername);
+  const [adminAvatarFile, __] = await knex("localfile")
+    .select("*")
+    .where("id", admin.avatarId);
+  // Deletes ALL existing entries and local files, except website owner
+  await knex("users").delete().whereNot("id", admin.id);
+  await knex("localfile").delete().whereNot("id", admin.avatarId);
+  const appDataPath = configService.get("TRANSCENDENCE_APP_DATA") as string;
   const avatarsPath = join(appDataPath, AVATARS_PATH);
-  rmSync(avatarsPath, {
-    recursive: true,
-    force: true,
-  });
+
+  try {
+    (await readdirSync(avatarsPath))
+      .filter((file) => file !== adminAvatarFile.filename)
+      .map((file) =>
+        unlinkSync(join(avatarsPath, file)));
+  } catch (error) {
+    throw error;
+  }
 
   const avatars = await Promise.all(
-    Array.from({ length: USERS_NUMBER }, createRandomAvatar),
+    Array.from({ length: USERS_NUMBER }, createRandomAvatar)
   );
   const users = avatars.map((avatar) => createRandomUser(avatar.id));
 
   // Inserts seed entries
-  await knex('localfile').insert(avatars);
-  await knex('users').insert(users);
+  await knex("localfile").insert(avatars);
+  await knex("users").insert(users);
 
   // Create users with default password
   const defaultAvatars = await Promise.all(
-    Array.from({ length: defaultUsernames.length }, createRandomAvatar),
+    Array.from({ length: defaultUsernames.length }, createRandomAvatar)
   );
   const usersWithPassword = await Promise.all(
     defaultUsernames.map((username, index) =>
-      createUserWithPassword(username, defaultAvatars[index].id),
-    ),
+      createUserWithPassword(username, defaultAvatars[index].id)
+    )
   );
 
   // Inserts seed entries
-  await knex('localfile').insert(defaultAvatars);
-  await knex('users').insert(usersWithPassword);
+  await knex("localfile").insert(defaultAvatars);
+  await knex("users").insert(usersWithPassword);
```

Comment:
my formatter now changes all strings to match this format. Does anyone know why?

Reply:
I think you're using WebStorm. Some of the answers from [here](https://stackoverflow.com/questions/49586279/making-webstorm-use-single-quotes-for-prettier) could help.

Reply:
done